setup.py

```diff
@@ -76,20 +74,22 @@ def getCondaNetCDF():
 
 # generate mint/__init__.py from mint/__init__.py.in
 init_file = ""
-with open(f"{DIR_NAME}/__init__.py.in") as fi:
+with open(f"{PACKAGE}/__init__.py.in") as fi:
     init_file = re.sub(r"@VERSION@", VERSION, fi.read())
-    with open(f"{DIR_NAME}/__init__.py", "w") as fo:
+    with open(f"{PACKAGE}/__init__.py", "w") as fo:
         fo.write(init_file)
 
 vtklib = getCondaVTK()
 nclib = getCondaNetCDF()
 
-# C++ 11 flag
-cpp11_flag = "-std=c++11"
-# give a chance to override the C++ 11 flag
 cpp_flags = os.getenv("CPPFLAGS")
-if cpp_flags:
-    cpp11_flag = cpp_flags  # on Windows: '/std:c11'
+cxx_flags = os.getenv("CXXFLAGS")
+if cxx_flags is not None:
+    extra_compile_args = cxx_flags.split()
+elif cpp_flags is not None:
+    extra_compile_args = cpp_flags.split()
+else:
+    extra_compile_args = ["-std=c++11"]
```

Comment:
Preference use of `CXXFLAGS` over `CPPFLAGS` over default of `-std=c++11`.

As we're building c++, this make sense given the standards.